fineract-provider/src/test/java/org/apache/fineract/common/RestAssuredFixture.java

```diff
@@ -33,16 +32,14 @@
  */
 public class RestAssuredFixture {
 
-    private final int httpPort;
+    private final int httpPort = 8443;
 
     private ResponseSpecification responseSpec;
     private RequestSpecification requestSpec;
 
-    public RestAssuredFixture(int httpPort) {
+    public RestAssuredFixture() {
         super();
-        this.httpPort = httpPort;
         Utils.initializeRESTAssured();
-        RestAssured.port = httpPort;
```

Comment:
I was quite surprised that you were able to remove this - how come? Is RestAssured's default 8443? Isn't it better to keep this explicit?

Reply:
RestAssured.port is a static field  hence picked up by StaticAssignmentInConstructor check. 
I removed  that line because Utils.initializeRESTAssured() sets the port to 8443.
I could try creating a method in Utils to initialize RestAssured with specific port or just revert this leave the check disabled.

Reply:
Does this makes it more explicit